target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch

```diff
@@ -0,0 +1,31 @@
+From 13e7bf2b3a6c00f2509513d508ab1b2a791aaf36 Mon Sep 17 00:00:00 2001
```

Comment:
I think this ID makes no sense, as it would refer on the kernel and you can't do this at the moment. So, maybe just drop this line again, though that's not really important.